src/commands/lambda/flare-command-validator.ts

```diff
@@ -0,0 +1,89 @@
+// Uses the Chain of Responsibility design pattern to validate the command
+// eslint-disable-next-line max-classes-per-file
+abstract class RequestHandler {
```

Comment:
Can we change the naming to something related to validation? Like `Validator`?

Also, what is a request? And what is the typing `{[key: string]: any}`?

Reply:
Renamed!
`request` is a bad name. It's given by passing in `this` from `flare.ts` -- it's an object with all of the variables `isDryRun`, `isInteractive`, `functions`, etc.

I have renamed `request` to `flags`

`flags = {
  isDryFun: false,
  isInteractive: false,
  ...
}`